definitions/divorce/json/CaseTypeTab.json

```diff
@@ -1174,6 +1174,48 @@
     "CaseFieldID": "ServiceSOTName",
     "TabFieldDisplayOrder": 111
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
it would be good to set `LiveFrom`, ones below, and a few in other Json files  to `now` date. 👍  But it is not a must if you want to have this merged ASAP :-)

Reply:
I changed the dates in the files where fields and events are created, and also their authorisations. In this particular one I just went along with the same date used for every line -  ultimately it should be easy to trace through the few files I mentioned above where the fields are created not at every single instance they are used.

Reply:
I see your point.